at_root/at_root_server/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: at_root_server
 description: Root server implementation.
-version: 1.0.0+1
+version: 1.0.0
```

Comment:
If we were at 1.0.0+1 then why are we going back to 1.0.0?

Reply:
Can i have it has 1.0.1?